src/shutdown/umount.c

```diff
@@ -387,8 +387,15 @@ static int delete_loopback(const char *device) {
         assert(device);
 
         fd = open(device, O_RDONLY|O_CLOEXEC);
-        if (fd < 0)
+        if (fd < 0) {
+                log_debug_errno(errno, "Failed to open loopback device %s: %m", device);
                 return errno == ENOENT ? 0 : -errno;
+        }
+
+        /* The loopback device deoesn't sync in-flight blocks when we clear the fd, hence sync explicitly
+         * first */
+        if (fsync(fd) < 0)
+                log_debug_errno(errno, "Failed to sync loop block device, ignoring %s: %m", device);
```

Comment:
`log_debug_errno(errno, "Failed to sync loop block device %s, ignoring: %m", device);` ?

Reply:
true!